linera-sdk/src/lib.rs

```diff
@@ -224,16 +224,16 @@ pub trait Contract: WithContractAbi + ContractAbi + Send + Sized {
         &mut self,
         authenticated: bool,
         application: ApplicationId<A>,
-        call: &A::ApplicationCall,
```

Comment:
This is intended: `ApplicationCall` -> `call`

Reply:
Can you change `argument: Self::ApplicationCall` into `call: Self::ApplicationCall` ? Thanks!